ui/src/pages/frontsite/PatronProfile/PatronCurrentLoans/components/LoanListEntry/LoanListEntry.js

```diff
@@ -39,10 +56,8 @@ export class LoanListEntry extends Component {
                 Loaned on {toShortDate(loan.metadata.start_date)}
               </Item.Meta>
               <Item.Description>
-                {}
-                You have extended this loan {
-                  loan.metadata.extension_count
-                } of {invenioConfig.loans.maxExtensionsCount} times
+                You have extended this loan {loan.metadata.extension_count} of{' '}
+                {invenioConfig.circulation.extensionsMaxCount} times
```

Comment:
In CDS this will be a very high number and it won't be easy to hide it. May I suggest to remove the second part `of ... times`?